CHANGELOG.md

```diff
@@ -1,5 +1,208 @@
 # Changelog
 
+## 5.0.0.rc1
+
+This is a major Teleport release with a focus on new features, functionality, and
+bug fixes. It’s a substantial release and users can review [5.0 closed issues](https://github.com/gravitational/teleport/milestone/39?closed=1)
+on Github for details of all items.
+
+#### New Features
+Teleport 5.0 is a major version and an introduction to Teleports Unified Access
+Plane. Going beyond just SSH access, with first class Kubernetes support
+and the new addition of Application Support.
+
+##### Teleport Application Access
+Teleport can be used to provide secure access to Applications. We've
+built this Teleport Application Access with an eye for securing those internal apps
+which might once of lived on the VPN or have a simple oauth mechanism but with little
+to no audit trail.  We've tested everything from router control panels to Single
+page javascript apps.
+
+Adding an Application follows the same UX as adding servers. An invite token needs
+to be created. This can be static or dynamic.
+
+```bash
+$ tctl tokens add --type=app
+```
+
+A Teleport node can be used to route to Application from the unified
+access plane. The Teleport node can be deployed alongside applications, enabling teams
+to setup access on a loopback address so only Teleport Application Access can access the app.
+
+```yaml
+# ...
+proxy_service:
+  # We've extended support for https certs. Teleport can now load multiple
+  # TLS certificates. In the below example we've obtained a wildcard cert
+  # that'll be used for proxy the applications.
```

Comment:
Tell more about how the certificate is selected (based on the hostname in the HTTPS request using SNI @russjones right?)